benchmark_runner/common/template_operations/templates/vdbench/internal_data/vdbench_vm_template.yaml

```diff
@@ -128,9 +128,9 @@ spec:
                 - export LOGS_DIR={{ LOGS_DIR }}
                 - echo @@~@@START-WORKLOAD@@~@@
                 {% if not scale -%}
-                - /vdbench/vdbench_runner.sh
+                - podman run --rm -e BLOCK_SIZES=$BLOCK_SIZES -e IO_OPERATION=$IO_OPERATION -e IO_THREADS=$IO_THREADS -e FILES_IO=$FILES_IO -e IO_RATE=$IO_RATE -e  MIX_PRECENTAGE=$MIX_PRECENTAGE -e DURATION=$DURATION -e  PAUSE=$PAUSE -e WARMUP=$WARMUP -e FILES_SELECTION=$FILES_SELECTION -e COMPRESSION_RATIO=$COMPRESSION_RATIO -e RUN_FILLUP=$RUN_FILLUP -e DIRECTORIES=$DIRECTORIES -e FILES_PER_DIRECTORY=$FILES_PER_DIRECTORY -e SIZE_PER_FILE=$SIZE_PER_FILE -e REDIS_HOST=$REDIS_HOST -e LOGS_DIR=$LOGS_DIR -e TIMEOUT=$TIMEOUT -v /workload:/workload --privileged {{ vdbench_image }} /vdbench/vdbench_runner.sh
                 {%- else -%}
-                - python3.9 /state_signals_responder.py $REDIS_HOST $WORKLOAD_METHOD $TIMEOUT
+                - podman run --rm -e BLOCK_SIZES=$BLOCK_SIZES -e IO_OPERATION=$IO_OPERATION -e IO_THREADS=$IO_THREADS -e FILES_IO=$FILES_IO -e IO_RATE=$IO_RATE -e  MIX_PRECENTAGE=$MIX_PRECENTAGE -e DURATION=$DURATION -e  PAUSE=$PAUSE -e WARMUP=$WARMUP -e FILES_SELECTION=$FILES_SELECTION -e COMPRESSION_RATIO=$COMPRESSION_RATIO -e RUN_FILLUP=$RUN_FILLUP -e DIRECTORIES=$DIRECTORIES -e FILES_PER_DIRECTORY=$FILES_PER_DIRECTORY -e SIZE_PER_FILE=$SIZE_PER_FILE -e REDIS_HOST=$REDIS_HOST -e LOGS_DIR=$LOGS_DIR -e TIMEOUT=$TIMEOUT -v /workload:/workload --privileged {{ vdbench_image }} python3.9 /state_signals_responder.py $REDIS_HOST $WORKLOAD_METHOD $TIMEOUT
```

Comment:
Quote all of the command line options and environment variables, e. g.

```
BLOCK_SIZES="$BLOCK_SIZES"
```
so they aren't inadvertently word-split if any of the values should happen to contain space characters.  You'll also need to update the golden files.

Reply:
All parameters are taken from [here](https://github.com/redhat-performance/benchmark-runner/blob/main/benchmark_runner/common/template_operations/templates/vdbench/vdbench_data_template.yaml)

Adding another double quotes can cause a problem and broke working workload

Reply:
All of those should be quoted.  What happens if someone inadvertently introduces a space into the data template, e. g.

```
IO_OPERATION: oltp1,oltp2,oltphw,odss2,odss128,read,read,read,read,write,write, write,write
```

Things will not function correctly if that happens.

Reply:
@RobertKrawitz, Fixed